### DISCUSSION TITLE
Is the DSC going to support the Trusted Security Profile?

### DISCUSSION THREAD

**Discussion by musi00** [International-Data-Spaces-Association/DataspaceConnector#114] [Q&A] [answered]
I've recently noticed some PRs that talk about integrating IDSCPv2 into the connector.  IDSCPv2 is the protocol that includes remote attestation.  Does that mean this connector will also support a Trusted security profile?  If so where does that leave the TrustedConnector repo (https://github.com/International-Data-Spaces-Association/trusted-connector)?

**Comment by juliapampus** [comment, marked as answer]
The DSC will be able to provide remote attestation and IDSCPv2, yes. However, the DSC is not part of the graduation scheme provided by the IDSA (see [here](https://github.com/International-Data-Spaces-Association/idsa/tree/main/graduation_scheme)). Thus, the DSC will never reach the offical level of IDS certification. I would highly recommend, to follow the developments of the Eclipse project (see [here](https://github.com/eclipse-dataspaceconnector/DataSpaceConnector)). This will replace the current DSC and aims at an IDS certification of a high trust level.

**Reply by musi00** [reply]
This is strange,  according to the documentation the DSC had already been granted IDS-Ready status:
https://international-data-spaces-association.github.io/DataspaceConnector/
https://www.isst.fraunhofer.de/de/news/pressemitteilungen/2020/Dataspace-Connector.html

Is IDS-Ready certification different than official certification??

**Reply by HeinrichPet** [reply]
It is: https://internationaldataspaces.org/the-ids_ready-label-is-an-excellent-opportunity-to-actively-prepare-for-ids-certification/

> In connection with IDS-ready, we assess a developer’s connector implementation concept, in order to verify, that the concept meets all IDS certification requirements for this component type (e.g. Connector) and selected security level (e.g. Trust). For the applicant, this means that they must develop an implementation concept in line with the IDS certification criteria, so that they can see what technical requirements their connector must live up to, and to what extent it does so already.
